draft-ietf-quic-transport.md

```diff
@@ -1477,35 +1481,240 @@ connection if the integrity check fails with a PROTOCOL_VIOLATION error code.
 
 ## Connection Migration {#migration}
```

Comment:
If this is client specific, should it be called Client Connection Migration, at least for the time being?

Reply:
...except that it isn't.  The title of the section is still "Connection Migration."